src/styles/transitions.js

```diff
@@ -65,3 +79,6 @@ export default {
     return Math.round(duration);
   },
 };
+
+export const easing = easingInternal;
```

Comment:
Can you add a comment? That would help avoiding people to import it in the source.